Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Pulling Ubuntu image
+FROM ubuntu:22.04
```

Comment:
I don't know about an entire Ubuntu image container.
Alpine maybe?

Reply:
I haven't done much with Alpine, but I like to watch it

Reply:
Hi, i has switch to Alpine based image